pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -576,6 +582,14 @@ The bridge-mode networking for UCR is identical to bridge mode networking for Do
 
 The only constraint in selecting an IPv4 subnet for `dcos_ucr_default_bridge_subnet` is that the subnet should not be used on the network to which the agents are connected. In other words, this subnet should be routeable from only within an agent.
 
+[enterprise]
+### feature_dcos_storage_enabled
+[/enterprise]
+
+Enables CSI support in the Mesos agents. Currently the only CSI plugin that is supported is the LVM plugin. The plugin allows Mesos to expose local volumes as resources to frameworks.
```

Comment:
Similar to above, i'd just mention that this flags will enable advanced storage support in DC/OS. I won't mention lvm at all here.